Tests/SentryTests/Integrations/Performance/UIViewController/SentryUIViewControllerSwizzlingTests.swift

```diff
@@ -85,7 +108,16 @@ class SentryUIViewControllerSwizzlingTests: XCTestCase {
         fixture.sut.start()
         let controller = TestViewController()
         controller.loadView()
+        let span = SentrySDK.span
+        
+        //To finish the transaction we need to finish `initialDisplay` span
+        //by calling `viewWillAppear` and reporting a new frame
+        controller.viewWillAppear(false)
+        Dynamic(SentryDependencyContainer.sharedInstance().framesTracker).reportNewFrame()
```

Comment:
Is this to cause `SentryTimeToDisplayTracker.framesTrackerHasNewFrame` to be called, ending the span it's managing? If so, please mention it here, or whatever other side effect is desired.

Reply:
So the reason I asked about `SentryTimeToDisplayTracker.framesTrackerHasNewFrame` is because when I looked at `SentryTimeToDisplayTracker.reportNewFrame` and clicked through to the delegate call, one of the options that comes up is `SentryTimeToDisplayTracker.framesTrackerHasNewFrame`:
![image](https://github.com/getsentry/sentry-cocoa/assets/3241469/59ac9df3-2b84-48be-b488-3ed928a4b6e8)

What is the actual codepath through which `SentryTimeToDisplayTracker.framesTrackerHasNewFrame` is called?

Reply:
Yes, you're right, I misread the class name you wrote. the call reportNewFrame() is a function of `SentryFramesTracker` that will call the `framesTrackerHasNewFrame` from its delegates.